Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 SUSE
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM opensuse/leap:15.2@sha256:7a6c8562ddbc367e20b08538b0ac6c57fc400a6edece39b8495afeaaa9cea58a AS staging
+RUN zypper refresh
+RUN zypper --non-interactive install \
+        git-core \
+        go1.14 \
```

Comment:
Why 1.14 in particular?  Since `KUBECTL_VERSION` is an arg, shouldn't this be one too (in case newer ones need newer go)?

Not sure there's a reasonable way to pick it out of [their configs](https://github.com/kubernetes/kubernetes/blob/v1.19.2/build/dependencies.yaml#L102).

Reply:
https://github.com/SUSE/kubecf-kubectl/pull/1/commits/206f06a26e973744a33ef7b709641a4bb7f69742 should do the magic. I tested with k8s v1.18.9 and v1.19.2. The former picks Go 1.13, the latter picks Go 1.15. I had to move to `opensuse/tumbleweed`, though, as `leap:15.2` doesn't provide Go 1.15. It should be no problem...